docs/index.html

```diff
@@ -152,16 +153,27 @@ <h1>Release Build Status</h1>
                                 .flatMap(item => {
                                     return item.runs.check_runs
                                         .filter(run => run.name.includes('Konflux'))
-                                        .map(run =>
-                                            `
+                                        .map(function (run) {
+
+                                            const parts = run.name.split(" / ")
+                                            const testName = parts[parts.length - 1]
+
+                                            let retestLink = `<a href="javascript:void(0);" onClick="retriggerTest('${item.org}', '${item.repo}', '${item.branch}', '${run.head_sha}', '${testName}')">/retest</a>`
+                                            if (parts.length === 3 && parts[1].endsWith("-ec")) {
+                                                retestLink = "<i title='retest not available on enterprise contract runs'>n/a</i>"
+                                            }
```

Comment:
I wouldn't bother filtering honestly as it couples this to the tests we have

Reply:
ok.

Reply:
Removed it